networkx/generators/random_graphs.py

```diff
@@ -704,7 +718,8 @@ def dual_barabasi_albert_graph(n, m1, m2, p, seed=None):
     Raises
     ------
     NetworkXError
-        If `m1` and `m2` do not satisfy ``1 <= m1,m2 < n`` or `p` does not satisfy ``0 <= p <= 1``.
+        If `m1` and `m2` do not satisfy ``1 <= m1,m2 < n``, `p` does not satisfy ``0 <= p <= 1``,
+        or the initial graph size m0 does not satisfy m1, m2 <= m0 <= n.
```

Comment:
Is `m0` the size of G or the order? Maybe "initial graph number of nodes m0 does not satisfy"...